### ISSUE TITLE
IQDB queries consistently errors out

### ISSUE THREAD

**Issue by BrokenEagle** [r888888888/danbooru#3358]
A few days ago I noticed that requests to ``/iqdb_queries`` would give 500 errors if more than a few requests were sent within any time period.  The following is the first portion of the stack trace returned.

```
HTTP error code: 500 Internal Server Error

/var/www/danbooru2/releases/20171101173557/app/logical/iqdb/download.rb:46:in `find_similar'
/var/www/danbooru2/releases/20171101173557/app/controllers/iqdb_queries_controller.rb:25:in `create_by_url'
/var/www/danbooru2/releases/20171101173557/app/controllers/iqdb_queries_controller.rb:11:in `index'
```

Are requests to IQDBS being throttled now, or is there something else going on?

**Comment by r888888888** [comment]
Not to my knowledge. The server does't appear to be under any load. I've restarted the unicorn processes; maybe one of them was hung.

**Comment by r888888888** [comment]
Actually looking at the logs what I see are a lot of failed connections to i.pximg.net (Name or service not known).

**Comment by BrokenEagle** [comment]
I first noticed it on image previews from Sankakucomplex so I thought it might be an issue with them, but then I saw it was also doing the same on image previews from Yandere.  I've been using that site to test the IQDB functionality since, so the issue wasn't just with Pixiv, although that site might have had a preponderance of the errors due to the numerous uploads from that site being checked on the uploads page.

However, I just retested it after you restarted unicorn and I'm not seeing any issues at the moment.
